tests/src/test/scala/cats/tests/StateTTests.scala

```diff
@@ -323,7 +323,9 @@ class IndexedStateTTests extends CatsSuite {
     implicit val G = ListWrapper.monad
     implicit val F = ListWrapper.alternative
     val SA = IndexedStateT.catsDataAlternativeForIndexedStateT[ListWrapper, Int](ListWrapper.monad, ListWrapper.alternative)
+
     checkAll("IndexedStateT[ListWrapper, Int, Int, Int]", AlternativeTests[IndexedStateT[ListWrapper, Int, Int, ?]](SA).monoidK[Int])
+    checkAll("IndexedStateT[ListWrapper, Int, Int, Int]", ApplicativeTests[IndexedStateT[ListWrapper, Int, Int, ?]](SA).applicative[Int, Int, Int])
```

Comment:
Hmm thanks. I wasn't aware of that. Was trying to solve the missing coverage for the `ap` override on the `Alternative` instance, but I now realize that this is pointless as there shouldn't be explicit overrides for `Applicative` methods for `StateT`. @djspiewak removed them in #1735.